### PR TITLE
DOC: Misc manual docs updates.

### PR DESCRIPTION
**Pull request by Carreau**
Those were autodetected but couldn't be automatically fixed.

### CODE REVIEW

File: scipy/signal/_spectral_py.py [hunk 1]
```diff
@@ -1650,6 +1650,7 @@ def _spectral_helper(x, y, fs=1.0, window='hann', nperseg=None, noverlap=None,
         segments, so that all of the signal is included in the output.
         Defaults to `False`. Padding occurs after boundary extension, if
         `boundary` is not `None`, and `padded` is `True`.
+
```
Comment (Carreau) [inline scipy/signal/_spectral_py.py:1653]: Without the blank line Return is seen by numpydoc as a parameter.

**Review by tupui — APPROVED** [review]
Great again @Carreau feel free if you find more 😅

File: scipy/stats/_stats_py.py [hunk 2]
```diff
@@ -5444,8 +5444,8 @@ def _mgc_stat(distx, disty):
 
     Parameters
     ----------
-    x, y : ndarray
-        `x` and `y` have shapes `(n, p)` and `(n, q)` or `(n, n)` and `(n, n)`
+    distx, disty : ndarray
+        `distx` and `disty` have shapes `(n, p)` and `(n, q)` or `(n, n)` and `(n, n)`
```
Comment (tupui) [inline scipy/stats/_stats_py.py:5448]:
```suggestion
        `distx` and `disty` have shapes `(n, p)` and `(n, q)` or
        `(n, n)` and `(n, n)`
```

**Review by tupui — APPROVED** [review]
Fixing the linter

**Pull request merged**